tests/integration/test_script_runner.py

```diff
@@ -25,476 +30,582 @@ class BackendDependencies(enum.Enum):
     SNOWFLAKE = "SNOWFLAKE"
 
 
+@dataclass
+class IntegrationTestFixture:
+    """IntegrationTestFixture
+
+    Configurations for integration tests are defined as IntegrationTestFixture dataclass objects.
+
+    Individual tests can also be run by setting the '-k' flag and referencing the name of test, like the following example:
+    pytest -v --docs-tests -m integration -k "test_docs[migration_guide_spark_v2_api]" tests/integration/test_script_runner.py
+
+    Args:
+        name (str): Name for integration test. Individual tests can be run by using the -k option and specifying the name of the test.
+        user_flow_script (str): Required script for integration test.
+        data_context_dir Optional[str]: Path of great_expectations/ that is used in the test.
+        data_dir Optional[str]: Folder that contains data used in the test.
+        extra_backend_dependencies Optional[BackendDependencies]: Optional flag allows you to tie an individual test with a BackendDependency. Allows for tests to be run / disabled using cli flags (like --aws which enables AWS integration tests).
+        other_files Optional[Tuple[Tuple[str, str]]]: other files (like credential information) to copy into the test environment.
+        util_script Optional[str]: Path of optional util script that is used in test script (for loading test_specific methods like load_data_into_test_database())
+        ge_requirement Optional[str]: allows you to set a specific version of GE to install. Default is current version of GE that is contained in the base directory (pip install .)
+    """
+
+    name: str
+    user_flow_script: str
+    data_context_dir: Optional[str] = None
+    data_dir: Optional[str] = None
+    extra_backend_dependencies: Optional[BackendDependencies] = None
+    other_files: Optional[Tuple[Tuple[str, str]]] = None
```

Comment:
`other_files` has a very particular structure; can we add a note in the docs about source/dest pairs and how that works?

Reply:
Great suggestion. Adding now